itou/api/README-FS-SSII.md

```diff
@@ -0,0 +1,194 @@
+
+- L'endpoint `api/v1/employee-records` n'est pas encore disponible mais en attendant vous pouvez déjà utiliser l'endpoint similaire `api/v1/dummy-employee-records` pour faire vos premiers tests.
+
+- L'endpoint `api/v1/dummy-employee-records` renvoit systématiquement 25 fiches salarié sur 2 pages avec des données factices mais réalistes. Peu importe l'utilisateur plateforme utilisé, donc vous pouvez simplement créer un compte candidat factice sur la plateforme [ici](https://emplois.inclusion.beta.gouv.fr/signup/job_seeker) et l'utiliser.
+
+## Points importants
+
+- Il y a au plus une FS par couple (SIRET, PassIAE). Autrement dit, si une même personne est recrutée pour la seconde fois (avec à chaque fois le même PassIAE) dans la même SIAE (ou deux structures ayant le même SIRET), cela ne donnera pas lieu à une nouvelle FS.
+
+- Les seules mesures concernées ici sont ACI, EI, ETTI et normalement les AI. Donc pas les EITI, GEIQ, EA, EATT.
+
+## Exemples techniques complets avec curl
+
+### Obtention d'un token
+
+Exemple :
+
+```
+curl -H 'Accept: application/json; indent=4' -d "username=john.doe@mail.com&password=abcdef" https://emplois.inclusion.beta.gouv.fr/api/v1/token-auth/
+```
+
+Réponse :
+
+```
+{
+    "token": "123abc123abc123abc123abc123abc123abc123abc"
+}
+```
+
+Notes :
+
+- Vous dévez échapper tout caractère spécial dans l'email et le mot de passe. Par exemple `+` devient `%2B`.
+- Pour les développeurs itou qui souhaitent utiliser l'API en local dev, remplacer `https://emplois.inclusion.beta.gouv.fr` par `http://127.0.0.1:8080`.
+
+
+### Obtention des FS
+
+Exemple :
+
+```
+curl -H 'Accept: application/json; indent=4' -H 'Authorization: Token 123abc123abc123abc123abc123abc123abc123abc' https://emplois.inclusion.beta.gouv.fr/api/v1/dummy-employee-records/
+```
+
+Réponse (première page présentant 20 des 25 FS) :
+
+```
+{
+    "count": 25,
+    "next": "https://emplois.inclusion.beta.gouv.fr/api/v1/dummy-employee-records/?page=2",
+    "previous": null,
+    "results": [
+        # Première FS.
+        {
+            "mesure": "ACI_DC",
+            "siret": "33055039315080",
+            [...]
+        },
+        # Seconde FS.
+        {
+            "mesure": "ACI_DC",
+            "siret": "33055039315080",
+            [...]
+        },
+        [...]
+    ]
+}
+```
+
+Obtention de la seconde page de résultats :
+
+```
+curl -H 'Accept: application/json; indent=4' -H 'Authorization: Token 123abc123abc123abc123abc123abc123abc123abc' https://emplois.inclusion.beta.gouv.fr/api/v1/dummy-employee-records/?page=2
+```
+
+## Référentiels utiles
+
+Tous les référentiels utiles mentionnés dans le JSON ci-dessous sont disponibles en CSV [ici](https://github.com/betagouv/itou/tree/vgrange/fiche_salarie/itou/fiche_salarie/management/commands/data).
```

Comment:
https://uxmovement.com/content/why-your-links-should-never-say-click-here/

Reply:
Ca fait plusieurs fois que je fais cette même boulette. Avec un peu de chance c'est la dernière! 🤞